packages/backend/src/services/MetricsExplorerService/MetricsExplorerService.ts

```diff
@@ -61,13 +68,148 @@ export class MetricsExplorerService<
         this.projectService = projectService;
     }
 
+    private async runComparePreviousPeriodMetricQuery(
+        user: SessionUser,
+        projectUuid: string,
+        exploreName: string,
+        metricQuery: MetricQuery,
+        timeDimensionConfig: TimeDimensionConfig,
+        dateRange: MetricExplorerDateRange,
+    ): Promise<{
+        rows: ResultRow[];
+        fields: Record<string, any>;
```

Comment:
can it something else from `any`?

Reply:
it should have been ItemMap. good catch!